ip_defs/sfp_panda_sync.tcl

```diff
@@ -52,3 +52,11 @@ CONFIG.gt0_val_clk_cor_seq_2_3 {00000000}               \
 CONFIG.gt0_val_clk_cor_seq_2_4 {00000000}               \
 ] [get_ips sfp_panda_sync]
 
+proc patch {BUILD_DIR} {
+    puts "Patching generated IP sources"
```

Comment:
am I right in thinking that this is needed to make the gtwizard IP compatible with the PICXO IP?

Reply:
Yes, that's right. We need to modify the sources that are generated by the IP. It may be possible to put this information directly into the IP tcl script, which would be neater, but I don't know.